analyzers/src/SonarAnalyzer.Common/SymbolicExecution/Constraints/ConstraintKind.cs

```diff
@@ -28,10 +28,14 @@ public enum ConstraintKind
     Null,
     NotNull,
     // Everything else
+    [Obsolete("Old SE engine only")]
```

Comment:
Are we 100% sure about these not being needed anywhere else? (I haven't checked).

Reply:
It's only used by the old SE engine. And I don't like the name: it's not about being constant or modified, but rather if it was filled up with cryptographically strong random numbers.

Reply:
I agree with the name in correlation to cryptography - but I do not remember if it was used also somewhere else that made sense.
If not - fine by me.